editoast/src/views/v2/train_schedule.rs

```diff
@@ -198,29 +186,26 @@ async fn get(
     Ok(Json(train_schedule.into()))
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, IntoParams)]
-struct GetBatchParams {
-    ids: Vec<i64>,
+#[derive(Debug, Deserialize, ToSchema)]
+struct BatchRequest {
+    ids: HashSet<i64>,
 }
 
 /// Return a specific train schedule
 #[utoipa::path(
     tag = "train_schedulev2",
-    params(
-        ("ids" = Vec<i64>, Query, description = "Ids of train schedule"),
-    ),
+    request_body = inline(BatchRequest),
     responses(
         (status = 200, description = "Retrieve a list of train schedule", body = Vec<TrainScheduleResult>)
     )
 )]
-#[get("")]
+#[post("")]
 async fn get_batch(
     db_pool: Data<DbConnectionPoolV2>,
-    params: QsQuery<GetBatchParams>,
+    data: Json<BatchRequest>,
```

Comment:
Do we still use QsQuery somewhere? If not `serde_qs` could be removed from `Cargo.toml`.

Reply:
We still use it in the path properties endpoint. We have a (limited) list of properties.

Reply:
Since it's the only usage we could probably add the props to the payload to remove the paradigm and the dependency altogether wdyt?